### PR TITLE
chore: remove undelegate method IN_ANGER

### LINKED ISSUE #7099: Remove `METHOD_IN_ANGER` from
https://github.com/vegaprotocol/vega/blob/e1cd037cef90aafe9b210841c22258f50addf8b5/protos/sources/vega/commands/v1/commands.proto

A party cannot undelegate by submitting a transaction IN_ANGER - an undelegation is 'in anger' if they remove their stake without undelegating. This METHOD is not required, so let's remove it.

# Tasks
- [ ] Remove `METHOD_IN_ANGER` from proto
- [ ] Remove corresponding
- [ ] Ensure it no longer appears in generated swagger

### PR DESCRIPTION
**Pull request by jeremyletang**
close #7099

### CODE REVIEW

**Comment by ze97286** [comment]
who was first? https://github.com/vegaprotocol/vega/pull/7101

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,21 +6,21 @@
 - [](https://github.com/vegaprotocol/vega/issues/xxxx) -
 
 ### 🗑️ Deprecation
-- [](https://github.com/vegaprotocol/vega/issues/xxxx) -
+- [7068](https://github.com/vegaprotocol/vega/issues/7068) - Alias `vegawallet info` to `vegawallet describe`, before definitive renaming.
```
Comment (ze97286) [inline CHANGELOG.md:9]: ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,21 +6,21 @@
...
 
 ### 🛠 Improvements
 - [7032](https://github.com/vegaprotocol/vega/issues/7032) - Make deposits and withdrawals `hypertables` and change `deposits_current` and `withdrawals_current` into views to improve resource usage
 
 ### 🐛 Fixes
-- [](https://github.com/vegaprotocol/vega/issues/xxxx) -
+- [7040](https://github.com/vegaprotocol/vega/issues/7040) - Block explorer use different codes than 500 on error
```
Comment (ze97286) [inline CHANGELOG.md:15]: ?

File: CHANGELOG.md (hunk 1 above)
Comment (jeremyletang) [reply CHANGELOG.md:9]: yea I moved them around because they were put on the wrong release, noticed it while adding the entries for this.

**Review by ze97286 — DISMISSED** [review]

**Review by guoguojin — APPROVED** [review]

**Pull request merged**